src/index.html

```diff
@@ -14,6 +14,7 @@
         <h1 class="txtMain">Awesome Books</h1>

```

Comment:
Everything works perfectly. Please wrap your elements in the `main` tag in order to follow the semantic rules. All the elements in the body should be wrapped in `header`, `main`, and `footer` tags.